test/lib/rule-finder.js

```diff
@@ -147,12 +170,12 @@ describe('rule-finder', function() {
     ])
   })
 
-  it('specifiedFile (absolut path) - current rules', function() {
+  it('specifiedFile (absolute path) - current rules', function() {
```

Comment:
Sorry to made you hope for a super editor plugin, but they just caught my eye :sweat_smile: